strings/edit_distance.py

```diff
@@ -0,0 +1,40 @@
+def edit_distance(source: str, target: str) -> int:
+    """
+    Edit distance algorithm is a string metric, i.e., it is a way of quantifying
+    how dissimilar two strings are to one another, that is measured by
+    counting the minimum number of operations required to transform one string
+    into another.
+    In genetic algorithms consisting of A,T, G, and C nucleotides, this matching
+    becomes essential in understanding the mutation in successive genes.
+    Hence, this algorithm comes in handy when we are trying to quantify the
+    mutations in successive generations.
+    Args:
+    source (type __string__): This is the source string, the initial string with
+    respect to which we are calculating the edit_distance for the target
+    target (type __string__): This is the target string, which is formed after n
+    number of operations performed on the source string.
+    Assumptions:
+    The cost of operations (insertion, deletion and subtraction) is all 1
+    Given two integers, return the sum.
+    :param source: str
+    :param target: str
+    :return: int
```

Comment:
```suggestion
    Edit distance algorithm is a string metric, i.e., it is a way of quantifying
    how dissimilar two strings are to one another. It is measured by
    counting the minimum number of operations required to transform one string
    into another.
    
    This implementation assumes that the cost of operations (insertion, deletion and
    substitution) is always 1
    
    Args:
    source: the initial string with respect to which we are calculating the edit distance
        for the target
    target: the target string, formed after performing n operations on the source string
```
Significantly cut down on the length of the docstring. The section on genetic algorithms is unnecessary because edit distance is not a genetic algorithm. It is a string algorithm that is _applied_ to problems in genetics.